Engine/src/sysFrame/Engine/Engine.cpp

```
@@ -96,3 +92,20 @@ void Engine::Finalize() {
+    engineDebug_ = true;
+
+    debugScene_ = std::make_shared<EngineDebug>();
+    debugScene_->Initialize();
+
+	System::Log(Log::Level::INFO, "DebugMode Enabled");
+    return;
+#endif
+    System::Log(Log::Level::ERR, "Request Cancelled!\nDebugMode is not available in Release Build.");
+}
+
+bool Engine::IsDebug() const {
```

cppcheck (2.10-2)

[style] The function 'IsDebug' is never used.

(unusedFunction)